easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.6-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,42 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'PythonPackage'
+
+name = 'bcbio-gff'
+version = '0.6.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/chapmanb/bcbb/tree/master/gff'
+
+description = """
+Read and write Generic Feature Format (GFF) with Biopython integration.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = [PYPI_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['74c6920c91ca18ed9cb872e9471c0be442dad143d8176345917eb1fefc86bc37']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+bcbiogffroot = 'lib/python%(pyshortver)s/site-packages'
```

Comment:
@robqiao please use use the `local_` prefix for these. See https://easybuild.readthedocs.io/en/latest/Easyconfig-files-local-variables.html for details - thanks.

Reply:
Thx @verdurin, local `--check-contrib` didn't pick up this one.

Reply:
Thanks.